prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1458,7 +1458,6 @@ int efa_rdm_ope_post_remote_write(struct efa_rdm_ope *ope)
 
 	assert(ope->iov_count > 0);
 	assert(ope->rma_iov_count > 0);
-	efa_rdm_ope_try_fill_desc(ope, 0, FI_WRITE);
```

Comment:
What's the reasoning for avoiding an MR reg call for zero byte write?

Reply:
`efa_rdm_ope_try_fill_desc()` calls `fi_mr_regv()` on the buffer.  This line assumes that the user will pass a real registered buffer to do a 0 byte write.  If you are doing a 0 byte write, then your buffer could be NULL, and your desc could be NULL b/c EFA shouldn't actually do anything with them.  The `fi_mr_regv()` will segfault on NULL desc.  Also segfaults for FI_INJECT which always gives a NULL desc.

Reply:
did you observe any performance impact for this commit?

Reply:
Looks like a speedup usec/xfer 1.30 (before), .98 (after) for
```
            "program": "/home/ec2-user/libfabric/fabtests/install/bin/fi_rma_bw",
            "args": ["-e", "rdm", "-o", "write", "-S", "10", "-p", "efa", "-E=9235"],
```